multichain-testing/tools/registry.ts

```diff
@@ -7,6 +7,10 @@ export const makeGetFile =
 
 type GetFilePathFn = ReturnType<typeof makeGetFile>;
 
+export type UseChainFn = Awaited<
+  ReturnType<ReturnType<typeof makeSetupRegistry>>
+>['useChain'];
+
```

Comment:
most of this is for:
```suggestion
export type MultichainRegistry = Awaited<
  ReturnType<ReturnType<typeof makeSetupRegistry>>
>;
```

Then instead of `UseChainFn`, `MultichainRegistry['useChain']`